### PR TITLE
Add README to archivematica-docs

### PR DESCRIPTION
**Pull request by sallain**
I've added a very basic README that has boilerplate information on the project, the license for the documentation, and instructions on where to go if users want to contribute to the docs. Any suggestions or additions are welcome!

### CODE REVIEW

**Review by sromkey — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Thank you for considering a contribution to the Archivematica documentation!
+For more information on contributing, please see the [Archivematica docs Github wiki](https://github.com/artefactual/archivematica-docs/wiki).
+This wiki describes the change submission process for the Archivematica docs and includes a style guide for new contributions.
+Following these guidelines helps us assess your changes faster and makes it easier for us to merge your submission.
+
+Note that some documentation lives in separate repos:
+
+* [Archivematica docs](https://github.com/artefactual/archivematica-docs): This repository! Contains the main user and administrator docs.
+* [Storage Service docs](https://github.com/artefactual/archivematica-storage-service-docs): Contains documentation for the Storage Service, Archivematica's back end.
+* [Format Policy Registry do](https://github.com/artefactual/archivematica-fpr-admin/tree/dev/docs): Contains documentation for the Format Policy Registry, which feeds Archivematica preservation planning rules and commands.
```
Comment (sevein) [inline README.md]: Typo: `s/do/docs`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+See the [LICENCE](LICENCE) file for details.
+
+# Contributing
+
+Thank you for considering a contribution to the Archivematica documentation!
+For more information on contributing, please see the [Archivematica docs Github wiki](https://github.com/artefactual/archivematica-docs/wiki).
+This wiki describes the change submission process for the Archivematica docs and includes a style guide for new contributions.
+Following these guidelines helps us assess your changes faster and makes it easier for us to merge your submission.
+
+Note that some documentation lives in separate repos:
```
Comment (ross-spencer) [inline README.md]: Suggest we use repository over its abbreviation for clarity and to reduce the barrier in understanding.

**Comment by ross-spencer** [comment]
I made one comment, but this (with the contributing guide) is great! Nice work @sallain.

**Review by kellyannewithane — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (sallain) [reply README.md]: Good call.

**Comment by sallain** [comment]
Resolves #85

**Pull request merged**